install.sh

```diff
@@ -353,15 +354,25 @@ function print_the_help {
 }
 
 UPGRADE=
-
+MACX=1
```

Comment:
Use consistent approach to flag variables: empty if unset; not empty when set; variable name is option name in uppercase.
```suggestion
NOX=1
```

Reply:
Done